File: client/bcephal-client-model/Sheets/Sheet.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Bcephal.Models.Sheets
{
    public class Sheet
    {

        public int Index { get; set; }
        public string Name { get; set; }

        public Sheet() { }

        public Sheet(int index, string name) : this()
        {
            this.Index = index;
            this.Name = name;
        }

        public override string ToString()
        {
            return this.Name;
        }

    }
}

File: server/bcephal-sheet/src/main/java/com/moriset/bcephal/sheet/domain/Sheet.java
/**
 * 
 */
package com.moriset.bcephal.sheet.domain;

import lombok.Data;

/**
 * @author Joseph Wambo
 *
 */
@Data
public class Sheet {

	private int index;

	private String name;

	private Sheet() {
	}

	private Sheet(int index, String name) {
		this();
		this.index = index;
		this.name = name;
	}

	@Override
	public String toString() {
		return this.name;
	}

}
